notifications_seed/Dockerfile.notifications-engine

```diff
@@ -0,0 +1,40 @@
+####
+# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
+###
+
+# Build the project
+FROM registry.access.redhat.com/ubi8/openjdk-17:latest AS build
+USER root
+COPY . /home/jboss
+WORKDIR /home/jboss
+RUN ./mvnw clean package -Dmaven.gitcommitid.skip=true -DskipTests --projects :checkstyle,:notifications-common,:notifications-common-aggregator,:notifications-database,:notifications-engine --no-transfer-progress
```

Comment:
We needed to edit notifications dockerfile, for adding this java system property `-Dmaven.gitcommitid.skip=true`  instead of altering the `pom.xml` file